metis-core/metis-core-rest/src/test/java/eu/europeana/metis/core/rest/TestDatasetController.java

```diff
@@ -615,6 +621,82 @@ public void getAllDatasetsByOrganizationNameInvalidUser() throws Exception {
         .getAllDatasetsByOrganizationName(any(MetisUser.class), anyString(), anyString());
   }
 
+  @Test
+  public void getDatasetsCountries() throws Exception {
+    MetisUser metisUser = TestObjectFactory.createMetisUser(TestObjectFactory.EMAIL);
+    when(authenticationClient.getUserByAccessTokenInHeader(TestObjectFactory.AUTHORIZATION_HEADER))
+        .thenReturn(metisUser);
+
+    MvcResult mvcResult = datasetControllerMock.perform(get("/datasets/countries")
+        .header("Authorization", TestObjectFactory.AUTHORIZATION_HEADER)
+        .contentType(TestUtils.APPLICATION_JSON_UTF8)
+        .content(""))
+        .andExpect(status().is(200))
+        .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8)).andReturn();
+
+    String resultListOfCountries = mvcResult.getResponse().getContentAsString();
+    Object document = Configuration.defaultConfiguration().jsonProvider()
+        .parse(resultListOfCountries);
+
+    List<Map<String, Object>> mapListOfCountries = JsonPath.read(document, "$[*]");
+    assertEquals(59, mapListOfCountries.size());
+    assertEquals(mapListOfCountries.get(22).get("enum"), "GREECE");
+    assertEquals(mapListOfCountries.get(22).get("name"), "Greece");
+    assertEquals(mapListOfCountries.get(22).get("isoCode"), "GR");
+  }
```

Comment:
Maybe assert that the size is equal to Country.values().size? That way if we change the countries the test will still hold. Similarly, instead of comparing to "GREECE" we could compare to Country.values()[22].name()? And the same for the other two lines.